packages/data-context/src/actions/CodegenActions.ts

```diff
@@ -152,7 +151,7 @@ export class CodegenActions {
     })
   }
 
-  getWizardFrameworkFromConfig (): WizardFrontendFramework | undefined {
```

Comment:
In general I think we should move away from naming things "Wizard". 🧙 are pretty cool, but it's probably a confusing, considering these are used throughout the entire app, not just in the Launchpad wizard.

We should just call them `ComponentFrontendFramework`, since that's what they are.

Reply:
I agree Wizards are pretty cool